packages/http-smoke-testing/README.md

```diff
@@ -124,6 +124,14 @@ All configured options will extend the values from default request data set (eve
 *Note: All three methods of [`RouteConfigCustomizer`](./src/RouteConfigCustomizer.php) accept `string $debugNote` as an argument.*
 *It is useful for describing the reasons of your configuration change because it may help you with debugging when the test fails.*
 
+To make smoke test configuration a little easier, you can use the annotations:
+
+- `@DataSet()`
+- `@Parameter(name="name", value="value")`
+- `@Skipped()`
+
+You can add them directly to your controller methods. See the example in `Shopsys\HttpSmokeTesting\Test\TestController`.
+
```

Comment:
Documentation should me much thorough so people know how to use it

Reply:
I'm quite a good with this. I just added link to example file.